assets/js/script.js

```diff
@@ -0,0 +1,199 @@
+// Assignment code
+var generateBtn = document.querySelector("#generate");
+
+// G L O B A L  A R R A Y S
+
+// arrays containing characters, symbols & numbers used in password generation
+
+// symbols
+const symbolCharacter = [
```

Comment:
think about your variable names and make sure they accurately describe what is actually stored in them. For example here, although it seems like a small thing, it would make sense to pluralise these (ie `symbolCharacters`), so that when we use it later in our code, we understand that we have multiple characters here, not just one.